core/src/main/java/com/yahoo/oak/EntryArrayInternal.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak;
+
+/**
+ * Abstraction for array of entries, where each entry has multiple fields (long).
+ */
+public interface EntryArrayInternal {
```

Comment:
I liked more `EntryArray` as a name for the interface and `EntryArrayInternal` as a name for implementation. Is it possible to swap between the names?